docs/cloud/netdata-functions.md

```diff
@@ -33,8 +33,9 @@ functions - [plugins.d](https://github.com/netdata/netdata/blob/master/collector
 | processes | Detailed information on the currently running processes on the node. | [apps.plugin](https://github.com/netdata/netdata/blob/master/collectors/apps.plugin/README.md) |
 
 If you have ideas or requests for other functions:
-* open a [Feature request](https://github.com/netdata/netdata-cloud/issues/new?assignees=&labels=feature+request%2Cneeds+triage&template=FEAT_REQUEST.yml&title=%5BFeat%5D%3A+) on Netdata Cloud repo
-* engage with our community on the [Netdata Discord server](https://discord.com/invite/mPZ6WZKKG2).
+* Open a [feature request](https://github.com/netdata/netdata-cloud/issues/new?assignees=&labels=feature+request%2Cneeds+triage&template=FEAT_REQUEST.yml&title=%5BFeat%5D%3A+) on Netdata Cloud repo
```

Comment:
```suggestion
* Participate in the relevant [GitHub discussion](https://github.com/netdata/netdata/discussions/14412)
* Open a [feature request](https://github.com/netdata/netdata-cloud/issues/new?assignees=&labels=feature+request%2Cneeds+triage&template=FEAT_REQUEST.yml&title=%5BFeat%5D%3A+) on Netdata Cloud repo
```

Reply:
I know it was not there before but since you are opening a PR on this taking the oportunity to add the Github discussion link 😁